diff --git a/drivers/net/wireless/Makefile b/drivers/net/wireless/Makefile
index e1c4141c6004..a78339800bdd 100644
--- a/drivers/net/wireless/Makefile
+++ b/drivers/net/wireless/Makefile
@@ -23,3 +23,10 @@ obj-$(CONFIG_WLAN_VENDOR_TI) += ti/
 obj-$(CONFIG_WLAN_VENDOR_ZYDAS) += zydas/
 
 obj-$(CONFIG_WLAN) += virtual/
+obj-$(CONFIG_RTL8189ES) += rtl8189es/
+obj-$(CONFIG_RTL8189FS) += rtl8189fs/
+obj-$(CONFIG_RTL8192EU) += rtl8192eu/
+obj-$(CONFIG_RTL8822EU) += rtl8822eu/
+obj-$(CONFIG_XRADIO) += xradio/
+obj-$(CONFIG_RTL8723DS) += rtl8723ds/
+obj-$(CONFIG_SPARD_WLAN_SUPPORT) += uwe5622/
