--- a/wfb_ng/services.py
+++ b/wfb_ng/services.py
@@ -133,7 +133,7 @@
                 fec_delay=cfg.fec_delay,
                 interleave_depth=interleave_depth,
                 link_id=link_id,
-                log_interval=settings.common.log_interval,
+                log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                 rcv_buf_size=settings.common.tx_rcv_buf_size,
                 snd_buf_size=settings.common.rx_snd_buf_size,
                 injection_retries=cfg.injection_retries,
@@ -178,7 +178,7 @@
                 key=os.path.join(settings.path.conf_dir, cfg.keypair),
                 rcv_buf_size=settings.common.tx_rcv_buf_size,
                 snd_buf_size=settings.common.rx_snd_buf_size,
-                log_interval=settings.common.log_interval,
+                log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                 link_id=link_id)).split() + (wlans if not is_cluster else [])

     df = RXProtocol(ant_sel_f, cmd, '%s rx' % (service_name,)).start()
@@ -262,7 +262,7 @@
                    key=os.path.join(settings.path.conf_dir, cfg.keypair),
                    rcv_buf_size=settings.common.tx_rcv_buf_size,
                    snd_buf_size=settings.common.rx_snd_buf_size,
-                   log_interval=settings.common.log_interval,
+                   log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                    link_id=link_id)).split() + (wlans if not is_cluster else [])

     tx_socket_path = '%s-tx-%s' % (service_name, os.urandom(4).hex())
@@ -293,7 +293,7 @@
                    fec_delay=cfg.fec_delay,
                    interleave_depth=interleave_depth,
                    link_id=link_id,
-                   log_interval=settings.common.log_interval,
+                   log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                    rcv_buf_size=settings.common.tx_rcv_buf_size,
                    snd_buf_size=settings.common.rx_snd_buf_size)).split() + wlans

@@ -379,7 +379,7 @@
                    key=os.path.join(settings.path.conf_dir, cfg.keypair),
                    rcv_buf_size=settings.common.tx_rcv_buf_size,
                    snd_buf_size=settings.common.rx_snd_buf_size,
-                   log_interval=settings.common.log_interval,
+                   log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                    link_id=link_id)).split() + (wlans if not is_cluster else [])

     tx_socket_path = '%s-tx-%s' % (service_name, os.urandom(4).hex())
@@ -410,7 +410,7 @@
                    fec_delay=cfg.fec_delay,
                    interleave_depth=interleave_depth,
                    link_id=link_id,
-                   log_interval=settings.common.log_interval,
+                   log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                    rcv_buf_size=settings.common.tx_rcv_buf_size,
                    snd_buf_size=settings.common.rx_snd_buf_size)).split() + wlans

@@ -501,7 +501,7 @@
                        key=os.path.join(settings.path.conf_dir, cfg.keypair),
                        rcv_buf_size=settings.common.tx_rcv_buf_size,
                        snd_buf_size=settings.common.rx_snd_buf_size,
-                       log_interval=settings.common.log_interval,
+                       log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                        link_id=link_id)).split() + (wlans if not is_cluster else [])

         log.msg('%s RX: %s' % (service_name, ' '.join(cmd_rx)))
@@ -536,7 +536,7 @@
                        fec_delay=cfg.fec_delay,
                        interleave_depth=interleave_depth,
                        link_id=link_id,
-                       log_interval=settings.common.log_interval,
+                       log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                        rcv_buf_size=settings.common.tx_rcv_buf_size,
                        snd_buf_size=settings.common.rx_snd_buf_size)).split() + wlans
         log.msg('%s TX: %s' % (service_name, ' '.join(cmd_tx)))
