--- a/wfb_ng/services.py
+++ b/wfb_ng/services.py
@@ -132,7 +132,7 @@
                 fec_timeout=cfg.fec_timeout,
                 fec_delay=cfg.fec_delay,
                 link_id=link_id,
-                log_interval=settings.common.log_interval,
+                log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                 rcv_buf_size=settings.common.tx_rcv_buf_size,
                 snd_buf_size=settings.common.rx_snd_buf_size,
                 injection_retries=cfg.injection_retries,
@@ -177,7 +177,7 @@
                 key_arg=key_arg(cfg),
                 rcv_buf_size=settings.common.tx_rcv_buf_size,
                 snd_buf_size=settings.common.rx_snd_buf_size,
-                log_interval=settings.common.log_interval,
+                log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                 link_id=link_id)).split() + (wlans if not is_cluster else [])
 
     df = RXProtocol(ant_sel_f, cmd, '%s rx' % (service_name,)).start()
@@ -261,7 +261,7 @@
                    key_arg=key_arg(cfg),
                    rcv_buf_size=settings.common.tx_rcv_buf_size,
                    snd_buf_size=settings.common.rx_snd_buf_size,
-                   log_interval=settings.common.log_interval,
+                   log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                    link_id=link_id)).split() + (wlans if not is_cluster else [])
 
     tx_socket_path = '%s-tx-%s' % (service_name, os.urandom(4).hex())
@@ -289,7 +289,7 @@
                    fec_timeout=cfg.fec_timeout,
                    fec_delay=cfg.fec_delay,
                    link_id=link_id,
-                   log_interval=settings.common.log_interval,
+                   log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                    rcv_buf_size=settings.common.tx_rcv_buf_size,
                    snd_buf_size=settings.common.rx_snd_buf_size)).split() + wlans
 
@@ -375,7 +375,7 @@
                    key_arg=key_arg(cfg),
                    rcv_buf_size=settings.common.tx_rcv_buf_size,
                    snd_buf_size=settings.common.rx_snd_buf_size,
-                   log_interval=settings.common.log_interval,
+                   log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                    link_id=link_id)).split() + (wlans if not is_cluster else [])
 
     tx_socket_path = '%s-tx-%s' % (service_name, os.urandom(4).hex())
@@ -403,7 +403,7 @@
                    fec_timeout=cfg.fec_timeout,
                    fec_delay=cfg.fec_delay,
                    link_id=link_id,
-                   log_interval=settings.common.log_interval,
+                   log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                    rcv_buf_size=settings.common.tx_rcv_buf_size,
                    snd_buf_size=settings.common.rx_snd_buf_size)).split() + wlans
 
@@ -494,7 +494,7 @@
                        key_arg=key_arg(cfg),
                        rcv_buf_size=settings.common.tx_rcv_buf_size,
                        snd_buf_size=settings.common.rx_snd_buf_size,
-                       log_interval=settings.common.log_interval,
+                       log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                        link_id=link_id)).split() + (wlans if not is_cluster else [])
 
         log.msg('%s RX: %s' % (service_name, ' '.join(cmd_rx)))
@@ -526,7 +526,7 @@
                        fec_timeout=cfg.fec_timeout,
                        fec_delay=cfg.fec_delay,
                        link_id=link_id,
-                       log_interval=settings.common.log_interval,
+                       log_interval=getattr(cfg, 'log_interval', settings.common.log_interval),
                        rcv_buf_size=settings.common.tx_rcv_buf_size,
                        snd_buf_size=settings.common.rx_snd_buf_size)).split() + wlans
         log.msg('%s TX: %s' % (service_name, ' '.join(cmd_tx)))
